README.md

```diff
@@ -48,82 +48,39 @@ is usually online from 8am-8pm GMT (London Time).
 
 # How to use localForage
 
-## Callbacks
+## Callbacks vs Promises
 
 Because localForage uses async storage, it has an async API.
 It's otherwise exactly the same as the
 [localStorage API](https://hacks.mozilla.org/2009/06/localstorage/).
 
-```javascript
-// In localStorage, we would do:
-var obj = { value: "hello world" };
-localStorage.setItem('key', JSON.stringify(obj));
-alert(obj.value);
-
-// With localForage, we use callbacks:
-localforage.setItem('key', obj, function(err, result) { alert(result.value); });
-```
+localForage has a dual API that allows you to either use Node-style callbacks 
+or [Promises](https://www.promisejs.org/). If you are unsure which one is right for you, it's recommend to use Promises.
 
-Similarly, please don't expect a return value from calls to
-`localforage.getItem()`. Instead, use a callback:
-
-```javascript
-// Synchronous; slower!
-var value = JSON.parse(localStorage.getItem('key'));
-alert(value);
+Here's an example of the Node-style callback form:
 
-// Async, fast, and non-blocking!
-localforage.getItem('key', function(err, value) { alert(value) });
-```
-
-Callbacks in localForage are Node-style (error argument first) since version
-`0.9.3`. This means if you're using callbacks, your code should look like this:
-
-```javascript
-// Use err as your first argument.
-localforage.getItem('key', function(err, value) {
-    if (err) {
-        console.error('Oh noes!');
-    } else {
-        alert(value);
-    }
+```js
+localforage.setItem('key', 'value', function (err) {
+  // if err is non-null, we got an error
+  localforage.getItem('key', function (err, value) {
+    // if err is non-null, we got an error. otherwise, value is the value 
+  });
 });
 ```
 
-You can store any type in localForage; you aren't limited to strings like in
-localStorage. Even if localStorage is your storage backend, localForage
-automatically does `JSON.parse()` and `JSON.stringify()` when getting/setting
```

Comment:
How about moving this paragraph right after [line 55](https://github.com/mozilla/localForage/pull/536/files#diff-04c6e90faac2675aa89e2176d2eec7d8R55)?